javascript/javascript1/week3/seriesTime.js

```diff
@@ -0,0 +1,24 @@
+const calculateTime = [
+    {
+      title: 'a',
+      days: 9, 
+      hours: 22,
+      minutes: 32,
+    },
+    {
+      title: 'b',
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: 'c',
+      days: 6,
+      hours: 8,
+      minutes: 0,
+    }
+  ]
+  for (let {title, days, hours, minutes} of calculateTime) {
```

Comment:
you're using advanced features we didn't cover in class, but OK if it works :)

Reply:
Yes, its working.